src/styles/block/card.scss

```
@@ -0,0 +1,70 @@
+.card {
+  width: 166px;
+  height: 358px;
```

The same as above, fix it everywhere
```suggestion
  width: 100%;
```

---

With such a condition, the element stretches across the entire width of the screen.